lib/browser/api/menu.js

```diff
@@ -164,10 +164,15 @@ Menu.buildFromTemplate = function (template) {
   const sorted = sortTemplate(filtered)
 
   sorted.forEach((item) => {
-    if (typeof item !== 'object') {
+    if (typeof item !== ('object' || null)) {
```

Comment:
The value of `('object' || null)` is `'object'`, so this is effectively `if (typeof item !== `object`)`

...which actually works as an is-object-or-null test because, due to ES logic, `typeof null` is also `'object'`.

But it would be simpler and clearer to write it that way.